src/main/java/org/apache/maven/shared/dependency/analyzer/DefaultProjectDependencyAnalyzer.java

```diff
@@ -189,6 +210,29 @@ else if ( file != null && file.isDirectory() )
         return artifactClassMap;
     }
 
+    protected Set<String> buildTestDependencyClasses( MavenProject project ) throws IOException
```

Comment:
can this be private or package protected?

Reply:
All the other buildMethod methods are protected so I made this protected to keep it the same

Reply:
It is very common to make methods overly open. Thus I wouldn't assume that the existing methods should have been protected. We can always make private methods more public if we discover a need. It is much harder to make a public method private again.